onboarding/mlsys/2-autodiff.md

```diff
@@ -0,0 +1,399 @@
+# Part 2: Autodifferentiation and Optimizers
+
+In "Part 1: Calculating Gradients and Averaging Them Together", we learned how
+gradient descent works mathematically.
```

Comment:
i would stick to either "you" or "we" throughout

Reply:
In "Part 1: Calculating Gradients and Averaging Them Together", you learned how
gradient descent works mathematically.